Mathlib/Computability/ContextFreeGrammar.lean

```diff
@@ -58,11 +58,9 @@ variable {N : Type*} {r : ContextFreeRule T N} {u v : List (Symbol T N)}
 
 /-- Inductive definition of a single application of a given context-free rule `r` to a string `u`;
 `r.Rewrites u v` means that the `r` sends `u` to `v` (there may be multiple such strings `v`). -/
-inductive Rewrites (r : ContextFreeRule T N) : List (Symbol T N) → List (Symbol T N) → Prop
-  /-- The replacement is at the start of the remaining string. -/
+inductive Rewrites (r : ContextFreeRule T N) : List (Symbol T N) → List (Symbol T N) → Prop/-- The replacement is at the start of the remaining string. -/
```

Comment:
Please revert this one (also for the next one)

Reply:
Done